shell/browser/api/electron_api_web_frame_main.cc

```diff
@@ -316,13 +348,39 @@ void WebFrameMain::Connect() {
   }
 }
 
+content::FindRequestManager* WebFrameMain::GetOrCreateFindRequestManager() {
```

Comment:
Need to check that the RFH is still alive.

```suggestion
content::FindRequestManager* WebFrameMain::GetOrCreateFindRequestManager() {
  if (!CheckRenderFrame())
    return nullptr;
```

Reply:
When does this happen ?

Reply:
In case a reference to the frame is held after its been destroyed.
```js
const { mainFrame } = browserWindow.webContents;
browserWindow.close();
mainFrame.findInFrame(...);
```

It can also occur upon navigation in some cases.